debug/image-on-globe.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Add a raster image to a map layer</title>
+    <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no">
+    <style>
+        body { margin: 0; padding: 0; }
+        #map { position: absolute; top: 0; bottom: 0; width: 100%; }
+        #globe { 
+            position: absolute;
+            top: 0; left: 0;
+            margin: 15px;
+            font-size: 14px;
+            padding: 8px;
+            background-color: lightblue;
+            font-weight: bold;
+        }
+    </style>
+</head>
+
+<body>
+<div id="map"></div>
+<button id='globe'>switch to globe</button>
+
+<script src='../dist/mapbox-gl-dev.js'></script>
+<script src='access_token_generated.js'></script>
+<script>
+    const map = new mapboxgl.Map({
+        container: 'map',
+        maxZoom: 7,
+        minZoom: 3,
+        zoom: 5,
+        center: [-75.789, 41.874],
+        style: 'mapbox://styles/mapbox/dark-v10'
+    });
+
+    document.getElementById('globe').addEventListener('click', (e) => {
+        if (e.target.innerHTML === 'switch to globe') {
+            e.target.innerHTML = 'switch to mercator';
+            map.setProjection('globe');
+        } else {
+            e.target.innerHTML = 'switch to globe';
+            map.setProjection('mercator');
```

Comment:
Instead of a globe/mercator switcher. Would it be preferable to introduce a projection dropdown? Similar to the one we have in the feature-state example?

<img width="381" alt="Screen Shot 2022-05-12 at 10 24 51 AM" src="https://user-images.githubusercontent.com/7061573/168133361-9a562af1-03a3-4669-bc95-f22499f389eb.png">

Reply:
A counterpoint is that testing all projections adds considerable time to the release testing process with relatively small improved chance of catching an issue. It's unlikely (but not impossible) for something to break in, e.g. Winkel Tripel but no other projection. Given the considerable unique codepaths and potential widespread usage of globe, I think it's the right call to test globe more heavily than other projections.

Of course, a projection dropdown doesn't mean that we necessarily need to test all projections every time. But it also adds a bit of time by being slower to navigate, particularly on mobile.

Reply:
Oops just saw your comment @SnailBones after I pushed the dropdown with the projections. I think a button that cycles between projections would be frustrating if you want to move back and forth between a few projections. Yes for the purpose of this launch we definitely want to focus on globe. Were there any previously resolved issues with raster image source of the other projections?

Reply:
Found a known bug with non-mercator projections and image source at low zoom levels (https://github.com/mapbox/mapbox-gl-js/issues/11238), I think it's good to keep the projections with this test since there has been a history of issues with image source being supported across all projections